caravel/forms.py

```diff
@@ -212,6 +212,14 @@ def __init__(self, viz):
                 'Y',
                 choices=self.choicify(datasource.column_names),
                 description="Columns to display"),
+            'druid_time_origin': SelectField(
+                'Origin',
+                choices=(
+                    ('now', 'now'),
+                    ('2016-04-10', 'even'),
```

Comment:
@mistercrunch Should this be `sunday`? Also are the choices set to be static? I'd imagine it be useful to input any day of the week as the anchor and based on the input the date can be set